src/components/BarChart/ChartContainer.tsx

```diff
@@ -7,9 +7,19 @@ type Props = {
   height: number
   data: ChartData[]
   onFocus: (data: ChartData | null) => void
+  showDays?: boolean
+  upColor?: string
+  downColor?: string
 }
 
-const ChartContainer = ({ height, data, onFocus }: Props) => {
+const ChartContainer = ({
+  height,
+  data,
+  onFocus,
+  showDays,
```

Comment:
might just recommend naming this `showXAxisLabel` or something along those lines since those labels could be different depending on the time scale

Reply:
Good point, let me update